src/TraceEvent/Parsers/ClrTraceEventParser.cs

```diff
@@ -2590,16 +2629,107 @@ public override object PayloadValue(int index)
             }
         }
 
-        private event Action<GenAwareBeginTraceData> m_target;
+        private event Action<GenAwareTemplateTraceData> m_target;
         #endregion
     }
-    public sealed class GenAwareEndTraceData : TraceEvent
+
+    public sealed class GCLOHCompactInfo
     {
-        public int Count { get { return GetInt32At(0); } }
-        public int ClrInstanceID { get { return GetInt16At(4); } }
+        /// <summary>
+        /// The time spent on planing the LOH for compaction
+        /// </summary>
+        public int TimePlan
+        {
+            get
+            {
+                return m_timePlan;
+
+            }
+        }
```

Comment:
nit formating, other places with a one liner getter all put them on one line like this -

`public int TimePlan { get { return m_timePlan; } }